code/game/objects/items/RPD.dm

```diff
@@ -70,6 +68,21 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 ))
 
 
+GLOBAL_LIST_INIT(pipe_paint_colors, list(
```

Comment:
These are used in more places than just this file, please keep it in the defines.